indico/modules/events/management/forms.py

```diff
@@ -270,6 +267,23 @@ def _init_visibility(self, event):
             self.visibility.warning = _("This event's visibility is currently limited by that of '{}'.").format(
                 real_horizon.title)
 
+    def validate_permissions(self, field):
+        entries = [{
+            'principal': principal_from_fossil(entry[0], allow_emails=True, allow_networks=True),
+            'permissions': entry[1]
+        } for entry in field.data]
+        for e in entries:
+            principal = e['principal']
+            permissions = e['permissions']
+            if isinstance(principal, IPNetworkGroup):
+                if permissions != ['access']:
+                    msg = _('IP Networks can only have "access" permission: {}:{}')
```

Comment:
"IP networks cannot have management permissions"

And I think showing both name and ID here is not very pretty - if it's the ID of the network, it's meaningless for a user.

Reply:
Also, if there are multiple placeholders in a translated string, *always* use the numbered or named version. In this particular case it may not matter much since `{}:{}` is meaningless enough so no translator would think about reordering it, but in other cases it may be necessary and for that the placeholders should be distinct and not use implicit positions. IMO the best choice is to always used named placeholders as soon as there's more than one in a translated string.